index.html

```diff
@@ -46,7 +45,6 @@ <h1 class="title">Build fast, responsive sites with WOW UI</h1>
         <!-- Hero Right Section Start -->
         <div class="hero-right">
             <div class="hero-img-cont">
```

Comment:
try to use better naming conventions for classes

Reply:
Okayy!! Will work on it